cmd/fluentbit-convert/main.go

```
@@ -0,0 +1,108 @@
+package main
+
+import (
+	"encoding/json"
+	"fmt"
+	"io"
+	"os"
+	"strings"
+
+	fluent "github.com/calyptia/go-fluentbit-config/v2"
+	"github.com/spf13/pflag"
+)
+
+func mustGetFormatFromExt(ext string) fluent.Format {
+	switch strings.ToLower(ext) {
+	case "json":
+		return fluent.FormatJSON
+	case "ini":
+		fallthrough
+	case "conf":
+		return fluent.FormatClassic
+	case "yml":
+		fallthrough
+	case "yaml":
+		return fluent.FormatYAML
+	}
+	panic(fmt.Errorf("unknown format extension: %s", ext))
```

nit: why not return an error to align with the other functions + their error handling? panic is rarely used, and would discourage using it for cases that can happen due to user input